posturebest/ContentView.swift

```diff
@@ -20,6 +19,11 @@ struct ContentView: View {
                 .tabItem {
                     Label("Configurations", systemImage: "gear")
                 }
+            
+            BluetoothDevicesView()
```

Comment:
Are we eventually moving the bluetooth to be in the profile tab? Or are we keeping it here?

Reply:
its a separate ticket